type.go

```diff
@@ -571,6 +668,33 @@ func (t fixedLenByteArrayType) Decode(dst encoding.Values, src []byte, enc encod
 	return encoding.DecodeFixedLenByteArray(dst, src, enc)
 }
 
+func (t fixedLenByteArrayType) AssignValue(dst reflect.Value, src Value) error {
+	v := src.ByteArray()
+	switch dst.Kind() {
+	case reflect.Array:
+		if dst.Type().Elem().Kind() == reflect.Uint8 && dst.Len() == len(v) {
+			// This code could be implemented as a call to reflect.Copy but
+			// it would require creating a reflect.Value from v which causes
+			// the heap allocation to pack the []byte value. To avoid this
+			// overhead we instead convert the reflect.Value holding the
+			// destination array into a byte slice which allows us to use
+			// a more efficient call to copy.
+			d := unsafe.Slice((*byte)(unsafecast.PointerOfValue(dst)), len(v))
+			copy(d, v)
+			return nil
+		}
+	case reflect.Slice:
+		if dst.Type().Elem().Kind() == reflect.Uint8 {
+			dst.SetBytes(copyBytes(v))
+			return nil
+		}
```

Comment:
Same suggestion as above:
```suggestion
		dst.SetBytes(copyBytes(v))
		return nil
```